Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Copy files",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=708389)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=708389)",
```

Comment:
Was this change intentional?

Reply:
It happened automatically when I built. Seemed harmless so I left it.